src/coprocessor/statistics/analyze.rs

```diff
@@ -346,6 +387,60 @@ impl<S: Snapshot> SampleBuilder<S> {
                 columns_info = &columns_info[1..];
             }
 
+            if self.analyze_common_handle {
+                // cur_val recording the current value's data and its counts when iterating index's rows.
+                // Once we met a new value, the old value will be pushed into the topn_heap to maintain the
+                // top-n information.
+                let mut cur_val: (u32, Vec<u8>) = (0, vec![]);
```

Comment:
a little bit confusing convention to use a tuple, could we use pattern matching instead?

Reply:
`topn_heap` uses the tuple as the element in it. I prefer to keep it.